lib/shelley/src/Cardano/Wallet/Shelley/Transaction/Size.hs

```diff
@@ -39,26 +37,21 @@ class MinSizeOf (t :: *) (n :: NetworkDiscriminant) (k :: Depth -> * -> *) where
 
 
 sizeOfSignedTx :: [TxIn] -> [TxOut] -> Int
-sizeOfSignedTx = error "sizeOfSignedTx to be implemented"
-
-
-instance MaxSizeOf Address 'Mainnet IcarusKey where
-    maxSizeOf = error "maxSizeOf Mainnet IcarusKey not implemented"
-instance MinSizeOf Address 'Mainnet IcarusKey where
-    minSizeOf = error "minSizeOf Mainnet IcarusKey not implemented"
-
-instance MaxSizeOf Address ('Testnet pm) IcarusKey where
-    maxSizeOf = error "maxSizeOf Testnet IcarusKey not implemented"
-instance MinSizeOf Address ('Testnet pm) IcarusKey where
-    minSizeOf = error "minSizeOf Testnet IcarusKey not implemented"
-
-instance MaxSizeOf Address 'Mainnet ByronKey where
-    maxSizeOf = error "maxSizeOf Mainnet ByronKey not implemented"
-instance MinSizeOf Address 'Mainnet ByronKey where
-    minSizeOf = error "minSizeOf Mainnet ByronKey not implemented"
-
-instance MaxSizeOf Address ('Testnet pm) ByronKey where
-    maxSizeOf = error "maxSizeOf Testnet ByronKey not implemented"
-instance MinSizeOf Address ('Testnet pm) ByronKey where
-    minSizeOf = error "maxSizeOf Testnet ByronKey not implemented"
-
+sizeOfSignedTx ins outs = 180 + (40 * length ins) + ((65 + fixme) * length outs)
+  where
+    -- According to the ledger specs, the 40 and 65 should be /the/ values, but
+    -- this integration test failure suggests otherwise:
+    -- ApplyTxError [LedgerFailure (UtxowFailure (UtxoFailure (FeeTooSmallUTxO
+    -- {pfUTXOminFee = Coin 553001000, pfUTXOgivenFee = Coin 520001000})))]\"}")
+    --
+    -- Maybe that value wasn't taking delegation keys into account?
+    --
+    -- Regardless:
+    -- TODO: Implement properly.
```

Comment:
The length of the output will actually depend on the value and the type of addresses used. We can assume worst-case scenario which are currently 65 bytes for addresses indeed. 32 bytes for just the value sounds huge.